### PR TITLE
launcher: update launcher UI

### PR DESCRIPTION
**Pull request by Muparadzi**
## Updates launcher UI in line with #230 

image
image

### CODE REVIEW

File: src/LauncherV3/MainWindow.xaml.cs [hunk 1]
```diff
@@ -156,4 +156,8 @@ private void locationButton_Click(object sender, RoutedEventArgs e)
 
     }
 
+    private void Beta_Checked(object sender, RoutedEventArgs e)
+    {
+
+    }
 }
```
Comment (namidaka) [inline src/LauncherV3/MainWindow.xaml.cs:163]: Discard

File: src/LauncherV3/MainViewModel.cs [hunk 2]
```diff
@@ -145,7 +149,8 @@ private bool CanUpdate()
     }
 
     public ICommand StartUpdateCrpgCommand => IsGameUpToDate ? StartCrpgCommand : UpdateGameFilesCommand;
-    public string StartUpdateText => IsGameUpToDate ? "Launch cRPG" : "Check For Update";
+    public string StartUpdateText => IsGameUpToDate ? "Launch cRPG" : "Update cRPG";
+    public LinearGradientBrush StartUpdateBackground => IsGameUpToDate ? new LinearGradientBrush(Color.FromArgb(50, 148, 56, 55), Color.FromArgb(255, 148, 56, 55), 90) : new LinearGradientBrush(Color.FromArgb(50, 251, 214, 1), Color.FromArgb(255, 251, 214, 1), 90);
```
Comment (namidaka) [inline src/LauncherV3/MainViewModel.cs:154]: MainViewModel should not have UI logic in MVVM pattern. 
This should be moved to XAML and binded to mainviewmodel
ChatGPT proposal 
```
<UserControl.Resources>
    <LinearGradientBrush x:Key="StartCrpgButtonBackground" StartPoint="0,0" EndPoint="1,1">
        <GradientStop Color="#328A28" Offset="0"/>
        <GradientStop Color="#205A18" Offset="1"/>
    </LinearGradientBrush>
    <LinearGradientBrush x:Key="UpdateCrpgButtonBackground" StartPoint="0,0" EndPoint="1,1">
        <GradientStop Color="#FBD602" Offset="0"/>
        <GradientStop Color="#B89500" Offset="1"/>
    </LinearGradientBrush>
</UserControl.Resources>

```

```
<Button Content="{Binding StartUpdateText}"
        Background="{StaticResource ResourceKey={Binding StartUpdateBackgroundResourceKey}}" />
```

**Review by namidaka — CHANGES_REQUESTED** [review]

File: src/LauncherV3/MainViewModel.cs (hunk 2 above)
Comment (Muparadzi) [reply src/LauncherV3/MainViewModel.cs]: I've moved the logic into the xaml.

File: src/LauncherV3/MainWindow.xaml.cs (hunk 1 above)
Comment (Muparadzi) [reply src/LauncherV3/MainWindow.xaml.cs]: Done

**Review by namidaka — APPROVED** [review]

**Pull request merged**